src/components/CommentsCard.jsx

```diff
@@ -0,0 +1,40 @@
+import { fetchCommentByArticleId } from "../../api"
+import {useEffect, useState} from 'react'
+import styles from '../css/CommentsCard.module.css'
+
+function CommentsCard (props){
+
+    const {article_id} = props
+
+    const [comments, setComments] = useState([])
+    const [isError, setIsError] = useState(false)
+
+    useEffect(()=>{
+        fetchCommentByArticleId(article_id).then((comments) => {
+            setComments(comments)
```

Comment:
consider doing this optimistically: https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering#optimistic-rendering